Meta/lint-ports.py

```diff
@@ -57,24 +57,64 @@ def read_port_dirs():
     return ports, all_good
 
 
+def check_package_files(ports):
+    """Check port package.sh file for required properties.
+
+    Args:
+        ports (set): List of all ports to check
+
+    Returns:
+        bool: no errors encountered
+    """
+
+    packages = set()
+    all_good = True
+    for port in ports:
+        package_file = port + '/package.sh'
+        if not os.path.exists(package_file):
+            continue
+        packages.add(package_file)
+
+    for package in packages:
+        with open(package, 'r') as fp:
+            data = fp.read()
+
+            if 'port=' not in data:
+                print('"Ports/{}" is missing "port" property'.format(package))
+                all_good = False
+
+            if 'version=' not in data:
+                print('"Ports/{}" is missing "version" property'.format(package))
+                all_good = False
+
+            if 'files=' not in data:
+                print('"Ports/{}" is missing "files" property'.format(package))
+                all_good = False
```

Comment:
Checking if these appear *anywhere* in the file seems error-prone, maybe we can iterate over individual lines and check if they *start with* one of the prefixes instead?

Reply:
Also, Python 3.5 had EOL and I don't think we need (or should) support it anymore - it's therefore safe to use f-strings rather than `.format()`.

Reply:
> 
> 
> Also, Python 3.5 had EOL and I don't think we need (or should) support it anymore - it's therefore safe to use f-strings rather than `.format()`.

The entirety of this file uses `format`. As do the other Python scripts:

```
$ grep -rn "format(" Meta/ | wc -l
24
```

And the contributing guidelines stipulate:

> Don't:
> Touch anything outside the stated scope of the PR.

Reply:
> 
> 
> Checking if these appear _anywhere_ in the file seems error-prone, maybe we can iterate over individual lines and check if they _start with_ one of the prefixes instead?

Resolved in #6380 without iterating over lines.